src/index.js

```diff
@@ -9,7 +9,3 @@ root.render(
     <App />
```

Comment:
As specified in the [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/be3b13a85ca425c0bf0eeb1f130b6a52e36deb4f/math-magicians/project_tests.md?plain=1#L31), kindly ensure that you write unit tests for all the react components. You seem not to have tested any component.

Also, as [required](https://github.com/microverseinc/curriculum-react-redux/blob/be3b13a85ca425c0bf0eeb1f130b6a52e36deb4f/math-magicians/project_tests.md?plain=1#L34), kindly ensure that user interaction is simulated.